cardano-crypto-praos/src/Cardano/Crypto/VRF/Praos.hs

```diff
@@ -322,31 +327,29 @@ proofFromBytes bs
         copyFromByteString ptr bs certSizeVRF
       return proof
 
-skFromBytes :: ByteString -> SignKey
-skFromBytes bs
-  | bsLen /= signKeySizeVRF
-  = error ("Invalid sk length " <> show @Int bsLen <> ", expecting " <> show @Int signKeySizeVRF)
-  | otherwise
-  = unsafePerformIO $ do
+skFromBytes :: D.Decoder s SignKey
```

Comment:
If we're changing this to a `Decoder`, perhaps we should rename it to `decodeSignKey` or `skFromCbor` or something like that.

Reply:
Fixed!